python-lib/odata_constants.py

```diff
@@ -4,21 +4,24 @@ class ODataConstants(object):
     DATA_CONTAINER_V2 = "d"
     DATA_RESULTS = "results"
     ENTITYSETS = "EntitySets"
+    FORMAT = "$format"
     INSTANCE = "odata_instance"
+    JSON = "json"
     LIST_TITLE = "odata_list_title"
     LOGIN = "sap-odata_login"
-    NEXT_LINK = "@odata.nextLink"
+    NEXT_LINK_V3 = "odata.nextLink"
+    NEXT_LINK_V4 = "@odata.nextLink"
```

Comment:
what's the difference between those?

Reply:
This is one example of the tiny differences between the v3 and v4 odata protocol. V3 sends you the last segment of the next page's URL using the "odata.nextLink"
<img width="383" alt="v3" src="https://user-images.githubusercontent.com/3763569/171854230-ffa875b8-5d99-4d61-bac0-ae6fc925cf92.png">

while v4 sends you the full URL using "@odata.nextLink" key...
<img width="467" alt="v4" src="https://user-images.githubusercontent.com/3763569/171854251-d92d92c3-6761-49ae-b62d-8e5bf36abbf4.png">